RMSwiftUI/RMSwiftUI/ContentView.swift

```diff
@@ -7,15 +7,158 @@
 
 import SwiftUI
 
+/// Экран с примерами работы Alert и ActionSheet
 struct ContentView: View {
+    @State private var isPresentedFirstAlert = false
+    @State private var isPresentedSecondAlert = false
+    @State private var isPresentedFirstActionSheet = false
+    @State private var isPresentedSecondActionSheet = false
+    @State private var isColorChanged = false
+    @State private var blockTwoText = Constants.blockTwoText
+    @State private var blockFourText =  Constants.blockFourText
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            VStack {
+                Text(Constants.blockOneText)
+                blockOne()
+            }.padding()
+            
+            VStack{
+                Text(blockTwoText)
+                blockTwo()
+            }.padding()
+            
+            VStack{
+                Text(Constants.blockThreeText)
+                blockThree()
+            }.padding()
+            
+            VStack {
+                Text(blockFourText)
+                    .multilineTextAlignment(.center)
+                    .foregroundColor(isColorChanged ? Color.red : Color.black)
+                blockFour()
+            }.padding()
+            
         }
-        .padding()
+    }
+    
+    fileprivate func blockOne() -> some View {
+        
+        return Button(
```

Comment:
```suggestion
        Button(
```
если  return не требует комплятор, писать его не надо